src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -26,7 +39,6 @@ md-progress-linear {
   }
 }
 
-.kd-deploy-input-row {
-  margin-bottom: $baseline-grid;
-  margin-top: $baseline-grid;
+.progress-or-message-container {
```

Comment:
Oh, I see! I'll check out this on Safari and Firefox.

Can you extract the variable `$bar-height: 2px;` to be in the file scope and reuse it here?